.gitignore

```diff
@@ -12,6 +12,7 @@
 *.suo
 *.trs
 *.user
+*.bak
```

Comment:
nit: try to maintain alphabetic order (I realize a few entries violate this rule already).